### PR TITLE
New JSDec home

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Since @wargio updated the [jsdec](https://github.com/rizinorg/jsdec) repository recently with some renaming, fixing it in Cutter too.

**Test plan (required)**

All green.

**Closing issues**

Closes https://github.com/rizinorg/cutter/issues/2549

### CODE REVIEW

**Comment by karliss** [comment]
Does it actually works now or does it still need changes on decompiler side? Can it now be renabled in CI builds?

**Comment by XVilka** [comment]
@karliss  I enabled JSDec in CI, seems to work.

**Comment by karliss** [comment]
Probably needs a separete issue created for making sure its packed correctly on macOS. I didn't do it during my last macOS packaging PR since it couldn't be compiled then.

**Comment by karliss** [comment]
Tested the windows build. jsdec is properly packaged and works in console widget but it is not listed in decompiler widget due to `e cmd.pdc` being removed. `JSDecDecompiler::isAvailable` needs to be adjusted.  Something like `cmdList("es").contains("jsdec")` is probably good enough as quick fix.

**Comment by XVilka** [comment]
@karliss just tested that package for Windows and `jsdec` seems to work out of the box.

**Review by wargio — APPROVED** [review]

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ jobs:
           APP_PREFIX=`pwd`/appdir/usr
           # export LD_LIBRARY_PATH=${APP_PREFIX}/lib:$Shiboken2_ROOT/lib
           export PATH=$PATH:${APP_PREFIX}/bin
-          # "../scripts/r2dec.sh" --prefix=`pwd`/appdir/usr
+          # "../scripts/jsdec.sh" --prefix=`pwd`/appdir/usr
```
Comment (karliss) [inline .github/workflows/ccpp.yml:145]: You can try uncommenting this to enable jsdec for appimage build.

**Review by karliss — CHANGES_REQUESTED** [review]

**Review by karliss — APPROVED** [review]

**Review by xarkes — APPROVED** [review]

**Pull request merged**